index.html

```diff
@@ -15,6 +15,21 @@
 </head>
 
 <body>
+  <script>
+  function brokenImage(link,id){
+    //alert("broken image!");
+    console.log(id+" image broken");
```

Comment:
Please remove, or is there a need to keep this in production for logging purposes?

Reply:
I used to keep this usually because it helps me in debugging the code later. 
I'd like to know how you deal with those sort of things.